### DISCUSSION TITLE
Having a capability to import Owners from a list

### DISCUSSION THREAD

**Discussion by danje57** [monarc-project/MonarcAppFO#411] [Ideas]
Hi all,

A great feature should be the capability to import risk owners from a file.
It could use user/role or function or a common shortname..

Than we could sort all risk per risk owner.

Regards

**Comment by ruslanbaidan** [comment]
Hi,

Could you explain in more details what did you have in mind by:
"It could use user/role or function or a common shortname.." ?

Risk owner is a free text field and can be populated with for ex. "First Name (DPO)" or anything that have meaning for you.
The field is searchable by typing and it's not possible to duplicate the same risk owner across an analysis.

We are considering to create a menu in the knowledge base where it would be possible to manage risk owners.

Adding a risk owner name to the risks sheet and sorting by it can be a good addition.

Regards

Ticket: https://github.com/monarc-project/MonarcAppFO/issues/402

**Reply by danje57** [reply]
This is exactly what I need!!! :)

Thx!
